README.md

```diff
@@ -271,3 +274,35 @@ ENTRYPOINT ["/go-binary"]
 ### Build final docker image
 
 `docker build -t docker-edgex-binary -f Dockerfile --build-arg BASE=base-build .`
+
+## Deployment On Sandbox
+
+
+The official site for Sandbox environment is **[https://jenkins.edgexfoundry.org/sandbox/](https://jenkins.edgexfoundry.org/sandbox/)**
+
+#### Get Access To Sandbox
+
+To deploy the pipeline on test environment, You need to request for access to sanbox through **[LF Service Desk](https://jira.linuxfoundation.org/servicedesk/customer/portals)**
```

Comment:
The official process to request access is found [on the lf-releng docs site](https://lf-releng-docs.readthedocs.io/en/latest/jenkins-sandbox.html)

Reply:
https://lf-releng-docs.readthedocs.io/en/latest/jenkins-sandbox.html#get-access-to-the-jenkins-sandbox

Reply:
Once the LF ID exists, the request can be made to add via JIRA ticket request.

Reply:
I will update these changes mentioned.